local.js

```diff
@@ -25,7 +25,12 @@ var config = module.exports.config = {};
  * Configuration namespace for the OAE tests.
  *
  * @param  {String}    timeout            The mocha timeout that should be used
+ * @param  {String}    level              The log level that should be used for testing
+ * @param  {String}    path               The log path that should be used for testing
  */
 config.test = {
-    'timeout': 60000
+    'timeout': 60000,
+    'level': process.env.OAE_TEST_LOG_LEVEL || 'info',
```

Comment:
These can just be `'info'` and `./tests.log` - they won't be set in `process.env` anymore.